sdk/iot/Azure.Iot.Hub.Service/src/ModulesClient.cs

```diff
@@ -35,17 +33,17 @@ protected ModulesClient()
         /// Initializes a new instance of DevicesClient.
         /// <param name="devicesRestClient"> The REST client to perform bulk operations on the module. </param>
         /// <param name="modulesRestClient"> The REST client to perform module and module twin operations. </param>
-        /// <param name="queryRestClient"> The REST client to perform query operations for the device. </param>
+        /// <param name="queryClient"> The convenience layer query client to perform query operations for the device. </param>
```

Comment:
Same as with the devices client. Taking in the query convenience layer helps to cut down on some code duplication we had